scipy/optimize/_slsqp_py.py

```diff
@@ -439,26 +457,57 @@ def cjac(x, *args):
 
             # Print the status of the current iterate if iprint > 2
             if iprint >= 2:
-                print("%5i %5i % 16.6E % 16.6E" % (majiter, sf.nfev,
-                                                   fx, linalg.norm(g)))
+                print(
+                    f"{majiter:5} {sf.nfev:5} {fx:16.6E} "
+                    f"{linalg.norm(g):16.6E}"
+                )
```

Comment:
Maybe a "good first issue" type thing would be for someone to add a test for this print--`codecov` saying it doesn't get hit, which isn't hugely surprising. Something like `pytest` `capsys` fixture is pretty good for capturing `stdout` and asserting on it.